test/sequential/test-module-loading.js

```diff
@@ -247,12 +247,12 @@ assert.deepEqual(children, {
 assert.throws(function() {
   console.error('require non-string');
   require({ foo: 'bar' });
-}, 'path must be a string or Buffer');
+}, /path must be a string/);
```

Comment:
Yes, but for this particular one, the message never included the `or Buffer`, that was a mis-edit that was missed when the buffers-in-fs pr was landed.